src/pkg/logger/example_logger_test.go

```diff
@@ -62,15 +63,24 @@ func Example_seed() {
 func Example_logger_standards() {
 	log := logger.Ctx(context.Background())
 
-	// 1. Keep messages short.  Lowercase text, no ending punctuation.
+	// 1. Keep messages short. When possible, messages should state the current action.
+	// Lowercase text, no ending punctuation.
 	// This ensures logs are easy to scan, and simple to grok.
 	//
 	// preferred
 	log.Info("getting item")
 	// avoid
 	log.Info("Getting one item from the service so that we can send it through the item feed.")
 
-	// 2. Do not fmt values into the message.  Use With() or -w() to add structured data.
+	// 2. Avoid statements like "unable to...", "failed to..", or "error when...".
+	// Error level logs automatically imply a failure to do the action.
+	//
+	// preferred
+	log.With("err", err).Error("connecting to repo")
+	// avoid
+	log.With("err", err).Error("unable to connect to repo")
```

Comment:
how about below messages? Are these fine?
1.  `drive id is empty`
2. `nil graph servicer`
3. `invalid refresh interval`